apps/marketing/src/app/(marketing)/layout.tsx

```diff
@@ -25,7 +25,7 @@ export default function MarketingLayout({ children }: MarketingLayoutProps) {
   }, []);
 
   return (
-    <div className="relative max-w-[100vw] overflow-y-auto overflow-x-hidden pt-20 md:pt-28">
+    <div className="relative max-w-[100vw] pt-20 md:pt-28">
```

Comment:
Fairly certain this will cause issues on Safari or Firefox when dealing with our `-inset-96` images

Reply:
Weird, `-inset` images look fine on my machine firefox/chrome/safari

What issues with images did you see?

This change resolves the side container sticky issue for SPM

Reply:
@Mythie 👶